standard-covoiturage_openapi.yaml

```diff
@@ -268,7 +268,7 @@ components:
           description: Carpooling duration in seconds.
         webUrl:
           type: string
-          description: URL of the journey on the webservice provider platform.
+          description: URL of the journey on the operator platform.
```

Comment:
@riclage @osarrat @adelcasse I am not sure we can go any further in this description. 

I tried something like "Depending on the  capabilities of the operator platform, calling this URL should  display the journey on the website or the mobile app of the  operator." but quickly,  it will have to summarise the whole deeplink experience (missing at least the "going through the app store" here). Seems to me like a cyclic dependency we would prefer to avoid ^^

Reply:
I'm OK with that.

Reply:
I guess given that both this parameter here and how the operator handles it in the booking flow is entirely in their control, we could leave it like that.